File: bubble-sort/bubble-sort.py
import random

def bubbleSort(array):
    length = len(array)
    for i in range(length):
        for j in range(length - i - 1):
            if array[j] > array[j + 1]:
                array[j], array[j + 1] = array[j + 1], array[j]

tab = []

for i in range(10):
    tab.append(random.randint(1, 100))
print(*tab, sep=", ")

bubbleSort(tab)

print(*tab, sep=", ")

File: bubble-sort/bubbleSort.java
import java.util.concurrent.ThreadLocalRandom;
import java.util.Arrays;

public class bubbleSort {
    public static void BubbleSort(int[] array){
        int n = array.length;
        
        for(int i = 0; i < n; i++){
            for(int j = 0; j < n - i - 1; j++){
                if(array[j] > array[j+1]){
                    int temp = array[j];
                    array[j] = array[j+1];
                    array[j+1] = temp;
                }
            }
        }
    }
    
    public static void main(String[] args) {
        int arraySize = ThreadLocalRandom.current().nextInt(1, 101);

        int[] tab = new int[arraySize];

        for(int k = 0; k < arraySize; k++){
            tab[k] = ThreadLocalRandom.current().nextInt(1, 101);
        }
        System.out.println("Przed sortowaniem:");
        System.out.println(Arrays.toString(tab));

        BubbleSort(tab);

        System.out.println("\nPo sortowaniu:");
        System.out.println(Arrays.toString(tab));
    }
    

}
